components/telemetry-operator/internal/webhook/logpipeline/validation/plugin_validator.go

```diff
@@ -76,28 +76,17 @@ func (pv *pluginValidator) validateOutput(pipeline *telemetryv1alpha1.LogPipelin
 	if err := validateHTTPOutput(pipeline.Spec.Output.HTTP); err != nil {
 		return err
 	}
-	if err := validateLokiOutPut(pipeline.Spec.Output.Loki); err != nil {
+	if err := validateLokiOutput(pipeline.Spec.Output.Loki); err != nil {
```

Comment:
We have an explicit output_validator. Why is output validated here? Should we move it?

Reply:
Done, see the last bullet point in the PR description.